pixi.toml

```diff
@@ -111,6 +111,7 @@ xgboost = "*"
 # =================== TESTS ===================
 # =============================================
 [feature.test-core.dependencies]
+cryptography = "*"
```

Comment:
Can you move this down to `[feature.test.dependencies]` and skip it for the core test?

Reply:
Maybe

Reply:
![image](https://github.com/user-attachments/assets/4f9a87c6-208d-46c2-86cb-11f0328cc687)